plugin-packs/postcss-preset-env/test/basic.ff66.expect.css

```diff
@@ -113,9 +107,7 @@
 
 .test-lab-function {
 	background-color: rgb(179, 35, 35);
-	background-color: color(display-p3 0.64331 0.19245 0.16771);
 	color: rgba(179, 34, 35, 0.5);
-	color: color(display-p3 0.64368 0.19188 0.16791 / 50%);
 	order: 17;
 }
```

Comment:
These colors were in gamut for `srgb`.
The fallback to `display-p3` was never needed.